docs/how-to-guides/develop-a-prompty/index.md

```diff
@@ -0,0 +1,384 @@
+# Prompty
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+Promptflow provides the `prompty` feature to provide customers with a simple way to develop prompt template.
```

Comment:
2 'provide' here, maybe asking copilot for some suggestions

Reply:
+1, maybe replace 2nd provide to enable

Reply:
copilot refines it to "Promptflow introduces the `prompty` feature, it is designed to simplify the development of prompt templates for customers."